pandas/core/window.py

```diff
@@ -173,6 +173,17 @@ def __getattr__(self, attr):
     def _dir_additions(self):
         return self.obj._dir_additions()
 
+    def _get_kwargs(self, **kwargs) -> Tuple[dict, dict]:
```

Comment:
Can you import `Dict` from typing and add subscription if possible?

Reply:
Only annotated separated [kwargs of window](https://github.com/pandas-dev/pandas/pull/27682/files#diff-813747abc5dedd9f437f30f692c84f48R923). The other one is remaining of kwargs from apply function which seems intractable. I guess we don't need to annotate empty dict nor it is possible in python